apps/egghead/src/lib/sanity/resource.ts

```
@@ -0,0 +1,204 @@
+
+			return await sanityWriteClient
+				.patch(sanityLesson?._id)
+				.set({
+					collaborators: [sanityInstructorReference],
+				})
+				.commit()
+
+		case 'course':
+			const sanityCourse = await getSanityCourseForEggheadCourseId(
+				post.fields.eggheadPlaylistId,
+			)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)